{
 "exception": "SHA256 mismatch: '3eff7b8a2c136304a91f6f7c3c5920af345af2075613bf030c7b4b53bb8996d9' != '7e84bb25a7be73d718942ec0cf0783b1281b5669461a3a26cb9dd8444e9208e8'",
 "traceback": [
  "Traceback (most recent call last):",
  "  File \"/home/runner/work/autotick-bot/autotick-bot/cf-scripts/conda_forge_tick/audit.py\", line 480, in main",
  "    deps = v[\"run\"](fctx, ctx)",
  "  File \"/home/runner/work/autotick-bot/autotick-bot/cf-scripts/conda_forge_tick/audit.py\", line 106, in depfinder_audit_feedstock",
  "    return extract_deps_from_source(recipe_dir)",
  "  File \"/home/runner/work/autotick-bot/autotick-bot/cf-scripts/conda_forge_tick/audit.py\", line 84, in extract_deps_from_source",
  "    cb_work_dir = _get_source_code(recipe_dir)",
  "  File \"/home/runner/work/autotick-bot/autotick-bot/cf-scripts/conda_forge_tick/utils.py\", line 640, in _get_source_code",
  "    return provide(md)",
  "  File \"/usr/share/miniconda/envs/run_env/lib/python3.8/site-packages/conda_build/source.py\", line 860, in provide",
  "    unpack(source_dict, src_dir, metadata.config.src_cache, recipe_path=metadata.path,",
  "  File \"/usr/share/miniconda/envs/run_env/lib/python3.8/site-packages/conda_build/source.py\", line 142, in unpack",
  "    src_path, unhashed_fn = download_to_cache(cache_folder, recipe_path, source_dict, verbose)",
  "  File \"/usr/share/miniconda/envs/run_env/lib/python3.8/site-packages/conda_build/source.py\", line 107, in download_to_cache",
  "    raise RuntimeError(\"%s mismatch: '%s' != '%s'\" %",
  "RuntimeError: SHA256 mismatch: '3eff7b8a2c136304a91f6f7c3c5920af345af2075613bf030c7b4b53bb8996d9' != '7e84bb25a7be73d718942ec0cf0783b1281b5669461a3a26cb9dd8444e9208e8'",
  ""
 ]
}